schedule/serializers.py

```diff
@@ -9,16 +9,17 @@ class DowntimeSerializer(serializers.ModelSerializer):
     site = serializers.ChoiceField(choices=configdb.get_site_tuples(), required=True)
     enclosure = serializers.ChoiceField(choices=configdb.get_enclosure_tuples(), required=True)
     telescope = serializers.ChoiceField(choices=configdb.get_telescope_tuples(), required=True)
+    instrument_type = serializers.ChoiceField(choices=configdb.get_instrument_type_tuples(include_blank=True), required=False)
 
     class Meta:
         model = Downtime
-        fields = ('start', 'end', 'site', 'enclosure', 'telescope', 'reason')
+        fields = ('start', 'end', 'site', 'enclosure', 'telescope', 'instrument_type', 'reason')
 
     def validate(self, data):
         if data['end'] <= data['start']:
             raise serializers.ValidationError(_("End time must be after start time"))
 
-        if not configdb.telescope_exists(data['site'], data['enclosure'], data['telescope']):
-            raise serializers.ValidationError(_('The site, enclosure, and telescope combination does not exist in Configdb'))
+        if not configdb.instrument_exists(data['site'], data['enclosure'], data['telescope'], data.get('instrument_type', '')):
```

Comment:
I think we should probably also check that a user entering a downtime via the admin interface cannot create a downtime for a resource that does not exist in configdb.

Reply:
Yeah, I guess I should add a form for the admin interface that makes the site, enclosure, telescope, and instrument_type a dropdown with only the values within configdb available as options.